docs/content.rst

```diff
@@ -575,7 +575,7 @@ Adding Automatic Naming for Content
 
 On some sites you don't want to set the name for every piece of content you
 create. Substance D provides support for this with a special kind of folder.
-You can configure your site to use the autonaming folder like by overrding
+You can configure your site to use the autonaming folder by overriding
 the standard folder:
 
```

Comment:
I don't agree with referencing the e-mail in the docs. It should have no more information than in the docs.